llama_index/download/dataset.py

```diff
@@ -0,0 +1,243 @@
+"""Download."""
+
+import json
+import os
+from pathlib import Path
+from typing import Any, Dict, List, Optional, Tuple, Union
+
+import requests
+import tqdm
+
+from llama_index.download.utils import (
+    initialize_directory,
+)
+
+LLAMA_DATASETS_CONTENTS_URL = (
+    f"https://raw.githubusercontent.com/run-llama/llama_datasets/main"
+)
+LLAMA_DATASETS_PATH = "/llama_datasets"
+LLAMA_DATASETS_URL = LLAMA_DATASETS_CONTENTS_URL + LLAMA_DATASETS_PATH
+
+LLAMA_DATASETS_LFS_URL = (
+    f"https://media.githubusercontent.com/media/run-llama/llama_datasets/main"
+)
+
+LLAMA_DATASETS_SOURCE_FILES_GITHUB_TREE_URL = (
+    "https://github.com/run-llama/llama_datasets/tree/main"
+)
+LLAMA_RAG_DATASET_FILENAME = "rag_dataset.json"
+LLAMA_SOURCE_FILES_PATH = "source_files"
+
+
+PATH_TYPE = Union[str, Path]
+
+
+def _get_file_content(loader_hub_url: str, path: str) -> Tuple[str, int]:
```

Comment:
why not just reuse functions from download_utils

Reply:
yea, i'll move these common functions into utils and remove the "_" prefix.